iothub/service/src/Query/QueryOptions.cs

```diff
@@ -12,5 +12,10 @@ public class QueryOptions
         /// The token to use for continuing the query enumeration
```

Comment:
Might need some nuance here, as in you won't need to use this if you just iterate items and don't care about pages.